README.md

```diff
@@ -68,6 +70,34 @@ iex> ExHal.follow_links(doc, "nonexistent")
 [{:error, %ExHal.Error{reason: "no such link"}}]
 ```
 
+### Actions
+
+If you want to take an action (ie, make a PUT, POST, etc) request you can do that too.
+
+```elixir
+name_change = """
+  { "name": "Bye!",
+    "_links": {
+       "self"   : { "href": "http://example.com" },
+       "profile": [{ "href": "http://example.com/special" },
+       { "href": "http://example.com/normal" }]
+    }
+  }
+  """
+
+# make a request that returns a HAL response
+iex> ExHal.put(doc, "self", name_change)
+{:ok, %ExHal.Document{...}}
+
+# make a request that just returns a response without without a body
+iex> {:ok, resp} = ExHal.post(doc, "add-child", "{\"name\": \"child\"}")
+{:ok, %ExHal.NonHalResponse{status_code: 201, headers: ["Location" => "http://example.com/child", ...], body: ""}}
```

Comment:
I think `:ok` is fine here, in this case it's not an error and the non-hal response is possible return value.  For context, our metric creation api takes a hal response but returns and empty body and a location header.